e2e/testdata/fn-eval/out-of-place-fnchain-stdout/.expected/config.yaml

```diff
@@ -9,51 +9,53 @@ stdOut: |
   apiVersion: config.kubernetes.io/v1alpha1
   kind: ResourceList
   items:
-    # Copyright 2021 Google LLC
-    #
-    # Licensed under the Apache License, Version 2.0 (the "License");
-    # you may not use this file except in compliance with the License.
-    # You may obtain a copy of the License at
-    #
-    #      http://www.apache.org/licenses/LICENSE-2.0
-    #
-    # Unless required by applicable law or agreed to in writing, software
-    # distributed under the License is distributed on an "AS IS" BASIS,
-    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    # See the License for the specific language governing permissions and
-    # limitations under the License.
-    - apiVersion: apps/v1
-      kind: Deployment
-      metadata:
-        name: nginx-deployment
-        namespace: staging
-        labels:
-          tier: backend
-        annotations:
-          config.kubernetes.io/index: "0"
-          config.kubernetes.io/path: resources.yaml
-          foo: bar
-      spec:
-        replicas: 3
-        selector:
-          matchLabels:
+  # Copyright 2021 Google LLC
```

Comment:
ResourceList is intermediate format. So indentation doesn't matter. Only final resources follow the indentation as per the annotation.

Reply:
My comment was more about understanding where is this indentation change coming from ?
Is it because we changed the default seqIndent to be compact vs wide ?

Reply:
Yes. ResourceList is marshalled with compact indentation as it is the default for kyaml.

Reply:
ok.